src/js/components/StatusIcon.js

```diff
@@ -0,0 +1,56 @@
+import React, { PropTypes, Component } from "react";
+
+import Icon from "#SRC/js/components/Icon";
+
+const ICONS_STATE = {
+  RUNNING: {
+    id: "circle-check",
+    color: "green"
+  },
+  TRANSITION: {
+    id: "spinner",
+    color: "grey"
+  },
+  STOPPED: {
+    id: "circle-minus",
+    color: "grey"
+  },
+  WARNING: {
+    id: "yield",
+    color: "yellow"
+  }
+};
+
+/**
+ * Status Icon is meant to be used across
+ * the application where it need to display any of the state below
+ * All green (RUNNING), transition (DEPLOYING, RECOVERING), Warning
+ *
+ * @class StatusIcon
+ * @extends {Component}
+ */
+class StatusIcon extends Component {
+  render() {
+    const state = this.props.state.toUpperCase();
```

Comment:
Please adjust so we don't transform the prop to uppercase - instead, be explicit about only accepting uppercase states which simplifies the component and provides a more explicit and clean API.